libc/docs/index.rst

```diff
@@ -12,7 +12,8 @@ The LLVM C Library
   LLVM-libc is not fully complete right now. Some programs may fail to build due
   to missing functions. If you would like to help us finish LLVM-libc, check
   out "`Contributing to the libc project <contributing.html>`__" in the sidebar
-  or ask on `discord <https://discord.com/channels/636084430946959380/636732994891284500>`__.
+  or ask on `discord <https://discord.com/channels/636084430946959380/636732994891284500>`__
+  (`invite link <https://discord.gg/xS7Z362>`__).
```

Comment:
Overall LGTM, but maybe this could be spelled more explicitly, something around the words "one need to join Discord using this invite link first" or something like this.

Reply:
How about "or ask on Discord (join via this invite link)" (obviously with the relevant links added!).